crates/router/src/core/recon.rs

```diff
@@ -1,100 +1,113 @@
 use api_models::recon as recon_api;
+#[cfg(feature = "email")]
 use common_utils::ext_traits::AsyncExt;
 use error_stack::ResultExt;
+#[cfg(feature = "email")]
 use masking::{ExposeInterface, PeekInterface, Secret};
 
+#[cfg(feature = "email")]
+use crate::{consts, services::email::types as email_types, types::domain};
 use crate::{
-    consts,
-    core::errors::{self, RouterResponse, UserErrors},
-    services::{api as service_api, authentication, email::types as email_types},
+    core::errors::{self, RouterResponse, UserErrors, UserResponse},
+    services::{api as service_api, authentication},
     types::{
         api::{self as api_types, enums},
-        domain, storage,
+        storage,
         transformers::ForeignTryFrom,
     },
     SessionState,
 };
 
+#[allow(unused_variables)]
```

Comment:
Do we need this annotation?

Reply:
Yes, this is needed as `auth_data` in this fn is used behind `email` feature only